File: leetcode/9. Palindrome Number/JAVA/src/palindromeNumber/PalindromeNumber.java
package palindromeNumber;

public class PalindromeNumber {
	public static void main(String[] args) {
		Solution solucion = new Solution();
		System.out.println(solucion.isPalindrome(121));
	}
}

class Solution {
    public boolean isPalindrome(int x) {
    	String entero = String.valueOf(x);
    	StringBuilder compuesto = new StringBuilder();
    	for(int i = entero.length()-1;i>=0;i--) {
    		compuesto.append(entero.charAt(i));
    	}
    	return compuesto.toString().equals(entero) ? true :  false;
    }
}

File: leetcode/9. Palindrome Number/JAVASCRIPT/PalindromeNumber.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    var entero = x.toString();
    var palindromo = "" ;
    for(var i = entero.length-1; i>=0;i--){
        var palindromo = palindromo + entero.charAt(i)
    }
    if (entero === palindromo){
        return true
    } else{
        return false
    }
};
